src/cpp/server/auth/ServerAuthHandler.cpp

```diff
@@ -373,6 +373,28 @@ void removeExpiredCookies()
    }
 }
 
+Error updateUser(const boost::shared_ptr<IConnection>& connection,
+                 const system::User& user)
+{
+   LOG_DEBUG_MESSAGE("Updating user in database: " + user.getUsername());
+
+   const std::string& username = user.getUsername();
+   Query updateQuery = connection->query("UPDATE licensed_users SET posix_name = :pn, display_name = :dn WHERE user_id = :uid")
```

Comment:
Since the `posix_name` column doesn't exist in open source, would it make more sense to make these changes in Pro?

For the future, we can make an effort to get this into the overlay code that Skye is introducing in https://github.com/rstudio/rstudio/pull/13610